src/mapInteractor.js

```diff
@@ -652,10 +652,9 @@ var mapInteractor = function (args) {
     m_touchHandler.lastEventType = evt.which;
     m_touchHandler.lastEvent = evt;
     /* convert touch events to have page locations */
-    var offset = $node.offset();
     if (evt.pageX === undefined && evt.center !== undefined && evt.center.x !== undefined) {
-      evt.pageX = evt.center.x + offset.left;
```

Comment:
We just talked about, but I'll add a note for the historical record: I was mistaken in adding the offset originally, but didn't notice until I added lines.  This fixed it.